styles.css

```diff
@@ -0,0 +1,886 @@
+    color: #ff5438;
+    padding: 0.6rem;
+    font-size: 12px;
+  }
+
+  .headline-section {
+    padding-top: 14%;
+  }
+
+  .pad-top {
+    margin: 0;
+  }
+
+  .speaker-details div {
+    margin: 0;
+  }
+
+  .mobile-cards > div {
+    display: none;
+  }
+
+  h1 {
+    font-size: 3.5rem;
+  }
+
+  .headline-section-about {
+    padding: 8rem 0;
+  }
+
+  .program-date {
+    padding: 0 15% 0 10%;
+  }
+
+  .main-title {
+    padding: 0 15% 0 10%;
+  }
+
+  .main-paragraph {
+    padding: 2rem 1rem;
+    margin-left: 10% !important;
+    width: 50%;
+  }
+
+  #program-section {
+    padding-bottom: 4rem;
+  }
+
+  .venue,
+  .main-paragraph,
+  #bold-text {
+    font-size: 1.2rem;
+  }
+
+  .footer-home {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    height: 10rem;
+  }
+
+  .program-list {
+    display: grid;
+    grid-template-columns: repeat(5, 20%);
+    row-gap: 2px;
+    padding: 2rem 10%;
+  }
+
+  .desktop-program-section-footer,
+  .desktop-cards {
+    display: flex;
+  }
+
+  .footer-about {
+    padding: 1rem 20%;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .program-list div {
+    flex-direction: column;
+    padding: 0.4rem !important;
+    margin: 0.06rem !important;
+  }
+
+  .speaker-details p {
+    margin: 0;
+  }
+
+  .program-list h3,
+  .program-list p {
+    width: 100% !important;
+  }
+
+  .speaker-details h3 {
+    margin: 0;
+  }
+
+  .program-description {
+    text-align: center;
+  }
+
+  .desktop-program-section-footer p {
+    text-decoration: underline;
+  }
+
+  .cards-container {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: repeat(3, auto);
+  }
+
+  .main-paragragh-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  .footer-about,
+  .footer-about div,
+  .footer-about h3 {
+    background-color: #2c2c2c !important;
+    color: white;
+  }
+
+  .about-main-title {
+    padding: 1rem 15%;
+  }
+
+  .about-main-paragraph {
+    width: 70%;
+    margin-left: 0;
+  }
+
+  .simple-cards {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin-top: 2rem;
+  }
+
+  .simple-cards div {
+    width: 35vw;
+  }
+
+  #special-mentions {
+    padding: 3rem 25% 6rem 25%;
+  }
+
+  #speakers-section {
+    padding: 1rem 12% 5rem 12%;
+  }
+
+  #see-past-header {
+    padding: 3rem 0;
+  }
+}
+
+/** Style for MOBILE */
+
+/* @media only screen and (max-width: 767px) {
+  .wrapper {
+    width: 375px;
+  }
+
+  .display-none,
+  .partners-home,
+  .footer-home,
+  .desktop-program-section-footer,
+  .footer-text,
+  .desktop-cards,
+  .pre-nav,
+  .pad-top {
+    display: none;
+  }
+
+  .menu-bar {
+    height: 0.35rem;
+    background-color: #6b6b69;
+    margin: 5px;
+    width: 2rem;
+    border-radius: 5px;
+  }
+  .sub-section-grey {
+    background-color: #414246;
+    color: white;
+  }
+} */

```

Comment:
- [ ] Kindly remove this commented lines of code to make your work neat and avoid confusion later on. 👍

Reply:
Done